### ISSUE TITLE
Dynamic parameter view interface

### ISSUE THREAD

**Issue by agrignard** [gama-platform/gama#2316]
Clearly not a top priority enhancement

I was just thinking that it could be nice (actually maybe it's already possible) to have a dynamic parameter view interface (show or not the parameter)

One simple example.

Imagine you have a parameter that enable to compute an interaction graph between agent and then you have a parameter that enable to change the distance parameter to compute the graph.

Typically when ComputeGraph is false you don't want to have the distance parameters visible because it will have no effect if you change it.

Now when I write

```
bool drawInteraction <- false parameter: "Draw Interaction:" category: "Visualization";
int distance parameter: 'distance ' category: "Visualization" min: 1 max:200 <- 100;
```

The user can play with the distance slider and it will have no effect on the model because draw interaction is set to false. 

However I don't know what could be a simple syntax in GAML

**Comment by AlexisDrogoul** [comment]
As it is already implemented in preferences, it shouldn't be too difficult to implement ...

**Comment by AlexisDrogoul** [comment]
Closed as a sort of duplicate of #2411, which is more general.
